astropy/uncertainty/distributions.py

```diff
@@ -151,8 +154,9 @@ def uniform(*, lower=None, upper=None, center=None, width=None, n_samples,
 
     Returns
     -------
-    distr : ``cls``, usually `Distribution`
+    distr : `Distribution` or object
```

Comment:
Should this be `~astropy.uncertainty.Distribution`?

Reply:
✅